desc/equilibrium/coords.py

```diff
@@ -105,7 +105,11 @@ def map_coordinates(  # noqa: C901
             f"don't have recipe to compute partial derivative {key}",
         )
 
-    profiles = get_profiles(inbasis + basis_derivs, eq)
+    profiles = (
+        kwargs["profiles"]
```

Comment:
This is the start of the fix that Rahul avoided by using the same technique I use for the coordinate mapping in #1335. It is not documented because it's not fully fixed yet, and the api may change etc.

Reply:
Is there an issue to link for this?